.github/workflows/testing.yml

```diff
@@ -98,14 +96,20 @@ jobs:
 
       # Only run vet for 'vet' runs.
       - name: Run vet.sh
+        env:
+          VET_SKIP_PROTO: 1
         if: matrix.type == 'vet'
         run: ./vet.sh -install && ./vet.sh
 
       # Main tests run for everything except when testing "extras"
       # (where we run a reduced set of tests).
       - name: Run tests
         if: matrix.type == 'tests'
+        env:
+          VET_BUILD_GO_PROTOC: 1
         run: |
+          ./vet.sh -install
+          protoc --version
```

Comment:
This doesn't want to be run for every version of Go that we test, presumably.

Let's test it in `extras` instead, below?

Reply:
sure, that's reasonable